viewer/packages/pointclouds/src/PotreeNodeWrapper.ts

```diff
@@ -82,16 +87,22 @@ export class PotreeNodeWrapper {
     return this._classification;
   }
 
-  pick(renderer: THREE.WebGLRenderer, camera: THREE.Camera, ray: THREE.Ray): PickPoint | null {
-    return this.octree.pick(renderer, camera, ray, { pickWindowSize: 20 });
+  get stylableObjectAnnotationIds(): Iterable<AnnotationMetadata> {
+    return this._annotations.map(a => {
+      return { annotationId: a.annotationId, assetId: a.assetId };
+    });
   }
 
-  get stylableObjects(): RawStylableObject[] {
-    return this._stylableObjects;
+  get stylableObjects(): PointCloudObjectAnnotation[] {
+    return this._annotations;
+  }
+
+  pick(renderer: THREE.WebGLRenderer, camera: THREE.Camera, ray: THREE.Ray): PickPoint | null {
+    return this.octree.pick(renderer, camera, ray, { pickWindowSize: 20 });
```

Comment:
There is no `pickWindowSize` anymore. Refer to @Savokr for new approach to point cloud picking - make sure we dont end up duplicating a lot of code here